build.gradle

```diff
@@ -165,8 +189,11 @@ def versions = [
   reformLogging   : '5.1.5',
   springBoot      : springBoot.class.package.implementationVersion,
   springfoxSwagger: '2.9.2',
+  serenity        : '2.2.12',
+  gradlePitest    : '1.5.1',
+  pitest          : '1.5.2',
```

Comment:
Are we using pitest in this project? If so is it configured in the nightly Jenkins pipeline?